server/routes/event.py

```diff
@@ -192,29 +192,19 @@ def event_node(dcid=DEFAULT_EVENT_DCID):
 
   subject_page_args = EMPTY_SUBJECT_PAGE_ARGS
   if place_dcid:
-    place_metadata = lib_subject_page_config.place_metadata(place_dcid)
+    place_metadata = lib_subject_page_config.place_metadata(
+        place_dcid, get_child_places=False)
     subject_config.metadata.contained_place_types.clear()
-    if place_metadata.contained_place_types_override:
-      subject_config.metadata.contained_place_types.update(
-          place_metadata.contained_place_types_override)
-    else:
-      subject_config.metadata.contained_place_types.update(
-          DEFAULT_CONTAINED_PLACE_TYPES)
-
-    place_type = None
-    config_place_types = subject_config.metadata.contained_place_types
-    for pt in place_metadata.place_types:
-      if pt in config_place_types:
-        place_type = pt
-        break
-    contained_place_type = config_place_types.get(
-        place_type, None) if place_type != None else None
+    subject_config.metadata.contained_place_types.update(
+        place_metadata.contained_place_types)
+    contained_place_type = place_metadata.contained_place_types.get(
+        place_metadata.place_type, None)
     subject_config = lib_subject_page_config.remove_empty_charts(
         subject_config, place_dcid, contained_place_type)
 
     # TODO: If not enough charts from the current place, add from the next place up and so on.
     subject_page_args = {
-        "place_type": json.dumps(place_metadata.place_types),
+        "place_type": json.dumps(place_metadata.place_type),
```

Comment:
nit: looks like place_type is just a string now right? so don't need to json.dumps

Reply:
thanks for catching that